library/psa_crypto_se.c

```diff
@@ -75,6 +90,123 @@ const psa_drv_se_t *psa_get_se_driver( psa_key_lifetime_t lifetime )
 
 
 
+/****************************************************************/
+/* Slot management */
+/****************************************************************/
+
+/** The type of bit vector elements. A bit vector is represented as an
+ * array of bit vector elements. */
+typedef unsigned bv_element_t;
+/* Number of bits per bit vector elements. Assumes no padding bits,
+ * which Mbed TLS checks in selftest.c. */
+#define BV_BITS_PER_ELEMENT ( sizeof( bv_element_t ) * CHAR_BIT )
+
+struct psa_drv_se_slot_usage_s
+{
+    size_t size; /** Number of elements of \c vector */
+    bv_element_t vector[1]; /* Should be vector[] but we don't allow C99 */
```

Comment:
We do allow C99. Use vector[]. CI won't break.

Reply:
Unfortunately, no, we don't allow C99 (I checked). We only allow the C99 features that Visual Studio 2010 supports. I don't think this includes `vector[]`: there's an error on line 205 in https://jenkins-internal.mbed.com/job/mbedtls-release-new/304/execution/node/319/log/
```
202     typedef struct
203     {
204         int foo;
205         int bar[];
206     } incomplete_t;
```

Reply:
Mistake on us to be using Visual Studio 2010 in our CI still, then.

Reply:
Moot because this chunk of code is no longer present.